source/crud/crud.rst

```diff
@@ -283,8 +283,8 @@ Read
      * SHOULD always send this value, if the cursor is not a TAILABLE_AWAIT cursor the server will
      * ignore it.
      *
-     * @note this option is an alias for `maxTimeMS`, used on `getMore` commands
-     * @note this option is not set on the `aggregate` command
+     * @note this option is an alias for maxTimeMS, used on getMore commands
+     * @note this option is not set on the aggregate command
```

Comment:
Note, this is unrelated to the spec. There is a separate commit in this PR to clean up backtick usage in the RST file.